cmd/root.go

```diff
@@ -150,18 +150,27 @@ func setFlags(cmd *cobra.Command, v *viper.Viper) {
 	flags.String(
 		"offline-queue-file",
 		"",
-		"(internal) Specify a offline queue file, which will be used instead of the default one.",
+		"(internal) Specify an offline queue file, which will be used instead of the default one.",
 	)
 	flags.String("plugin", "", "Optional text editor plugin name and version for User-Agent header.")
 	flags.String("project", "", "Override auto-detected project."+
 		" Use --alternate-project to supply a fallback project if one can't be auto-detected.")
+	flags.String(
+		"project-folder",
+		"",
+		"Optional workspace path. Override auto-detected project folder."+
+			" Useful only when --relative-paths is true.")
```

Comment:
Optional workspace path. Only used when `relative_paths=true` in the config file, or `--relative-paths` flag is present.